yhondanilo/chess.rb

```diff
@@ -0,0 +1,48 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila = fila
+    @columna = columna
+    intercalar
+  end
+  def intercalar
```

Comment:
Assignment Branch Condition size for intercalar is too high. [33/15]<br>Use empty lines between defs.